### DISCUSSION TITLE
Adjustment in TradeManager

### DISCUSSION THREAD

**Discussion by TLable** [MicroTrendsLtd/NinjaTrader8#81] [General]
Are there any simple Brushes to control the TradeManager Brush in cs?
I found one in a text file but any changes to that color name 'GrayTextBrushKey' results in no TradeManager.

**Comment by MicroTrendsTom** [comment]
is this it?
https://github.com/MicroTrendsLtd/NinjaTrader8/blob/main/ATSQuadroStrategyBase/NinjaTrader%208/bin/Custom/Strategies/ATSQuadroStategyBase/ATSQuadroStrategyBaseTradingUILayerBase.Properties.cs
